data/LemmaCountVectorizer.py

```diff
@@ -0,0 +1,10 @@
+from nltk import WordNetLemmatizer
```

Comment:
Name of this file: please follow the naming convention of all other files please (should be `lemma_count_vectorizer.py`)